HACKING.md

```diff
@@ -99,6 +99,10 @@ Such build image can be run the same way
 
     $ make run_db
 
+Find out the database container's IP address:
+
+    $ docker inspect db | grep -m 1 '\<IPAddress'
```

Comment:
https://docs.docker.com/engine/reference/commandline/inspect/ as an alternative.

Reply:
Fixed.

Reply:
You didn’t *have to* :D I believe that might result in more than one address being output in some cases as well, if I’m not mistaken (`-m1` in grep would help guard against that).